Errors when running on Go 1.23 projects

I tried running this standalone today, after seeing no errors reported in golangci on a repository where they should be reported.

```
 sloglint -attr-only true -no-global all ./...
-: This application uses version go1.22 of the source-processing packages but runs version go1.23 of 'go list'. It may fail to process source files that rely on newer language features. If so, rebuild the application using a newer version of Go.
./document_client.go:1:1: package requires newer Go version go1.23
...
sloglint: 8 errors during loading

```

```
 sloglint -V
sloglint version 0.7.2 linux/amd64
go version go1.23.3 linux/amd64
```

Not really sure what that message really indicates. Is it just my setup?

--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module go-simpler.org/sloglint
 
-go 1.22.0
+go 1.23
 
 require (
 	github.com/ettle/strcase v0.2.0

